Add external key manager configuration docs for Choreo Connect (#3769)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/configuring-an-external-key-manager.md

# Configuring an External Key Manager

> WSO2 Choreo Connect provides the capability to configure external key managers through APIM admin portal. The issuer data is retrieved from the event hub at the startup and updated as the changes are made from the admin portal.

Reviewer: ```suggestion
Choreo Connect provides the capability to configure external key managers through APIM admin portal. The issuer data is retrieved from the event hub at the startup and updated as the changes are made from the admin portal.
```

Author: Added with https://github.com/wso2/docs-apim/pull/3769/commits/858d270db004a2d3a629254b6f85eb12be424c17

### Step 1 - Configure Choreo Connect with API Manager

> Please refer [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/#configure-choreo-connect-with-api-manager).

Reviewer: ```suggestion
Please refer [configuring Choreo Connect with API Manager]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/#configure-choreo-connect-with-api-manager).
```

Author: Added with https://github.com/wso2/docs-apim/pull/3769/commits/858d270db004a2d3a629254b6f85eb12be424c17

### Step 2 - Add the external key manager to API Manager

> Please refer [here]({{base_path}}/administer/key-managers/overview) to configure the desired key manager. 

Reviewer: ```suggestion
Please refer [Multiple Key Manager Support in WSO2 API Manager]({{base_path}}/administer/key-managers/overview) to configure the desired key manager. 
```

Author: Added with https://github.com/wso2/docs-apim/pull/3769/commits/858d270db004a2d3a629254b6f85eb12be424c17
